pkg/netutils/ips.go

```diff
@@ -0,0 +1,53 @@
+package netutils
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// GetDefaultIPNet returns the default interface for the node, and the subnet mask for that node, using the same logic as k0s
```

Comment:
Could you please include a perm link as a reference to where this logic lives in k0s?